genestack_client/genestack_shell.py

```diff
@@ -314,21 +318,22 @@ def process_command(self, command, argument_list, shell=False):
         try:
             args = p.parse_args(argument_list)
         except SystemExit:
-            return
+            return 1
         command.set_connection(self.connection)
         command.set_arguments(args)
         try:
-            return command.run()
+            command.run()
```

Comment:
We don't need them.   This make application is less complex.   Docstring to run added.